SleepEngine/Demos/Bubbles/Entities/Bubble.cpp

```diff
@@ -19,7 +19,7 @@ void Bubble::update(float dt)
 {
     auto& transform = slp::getTransform(m_object);
 
-    float const distanceFlown = (transform.getPosition() - m_startPosition).length();
+    float const distanceFlown = static_cast <float>((transform.getPosition() - m_startPosition).length());

```

Comment:
so what's the type of lenth?

Reply:
glm::high_bla_bla_bla <2, float>::lengh_t